main.go

```diff
@@ -113,6 +114,13 @@ func main() {
 			Category:    "Connectivity:",
 			Destination: &rpcPort,
 		}),
+		altsrc.NewIntFlag(&cli.IntFlag{
+			Name:        GUI_PORT,
+			Usage:       "Specifies the tcp port for the Nitro Connect GUI.",
+			Value:       5005,
+			Category:    "Connectivity:",
```

Comment:
Minor side issue:

Do these categories ever get used?
- yes: extract into constants
- no: maybe scrub, replace with empty string or `"categoriesnotused"` const

Reply:
They do get used in the "help" function I believe, to organise the output. Good call on constants.